src/app/api/common/proposals/getProposals.ts

```diff
@@ -114,14 +114,19 @@ async function getProposal(proposalId: string) {
 async function getProposalTypes() {
   const { namespace, contracts } = Tenant.current();
 
-  return prisma[`${namespace}ProposalTypes`].findMany({
-    where: {
-      contract: contracts.proposalTypesConfigurator!.address,
-      name: {
-        not: "",
+  try {
+    const results = await prisma[`${namespace}ProposalTypes`].findMany({
+      where: {
+        contract: contracts.proposalTypesConfigurator!.address,
+        name: {
+          not: "",
+        },
       },
-    },
-  });
+    });
+    return results;
+  } catch (error) {
+    return [];
+  }
```

Comment:
Hm, actually, I think this happens if the query errors, and this way the UI doesn't brick if it errors. But if it errors it means we should probably catch the error and show UI accordingly rather than hiding it with empty array. I can pull it out.